Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,64 @@
+public class View: Viewable {
+	var io: Receivable
+	var guess: String!
+
+	public init(io: Receivable) {
+		self.io = io
+	}
+
+	public func receiveGuess(getGuess: () -> String) -> String {
+		guess = String(getGuess())
+		while (guess == "" || !(containsOnlyLetters(guess))) {
+			io.display("Invalid guess. Please enter a lowercase letter or word.")
+			guess = String(getGuessFromUser())
+			print(guess)
+		} 
+		return guess
+	}
+
+	public func getGuessFromUser() -> String {
+		return io.getUserInput(io.myReadLine)
```

Comment:
This was overly complicated in my `receiveGuess` dance. I changed this to just return `io.getUserInput()`